gulpfile.js

```diff
@@ -0,0 +1,12 @@
+import gulp from 'gulp';
+
+import { watcher, reset, htmlCopy, server, scss, js } from "./gulp/tasks/functions.js";
+
+const mainTasks = gulp.parallel(htmlCopy, scss, js);
```

Comment:
const mainTasks = gulp.parallel(htmlCopy, scss, js); - do we need this?

Those tasks will be run by secondTasks that you added in the next row.